Automation/S3/delete_s3_buckets.py

```diff
@@ -0,0 +1,194 @@
+import boto3
+from botocore.config import Config
+from botocore.exceptions import ClientError, NoCredentialsError
+from colorama import init, Fore
+import argparse
+
+COL_SIZE = 15
+init(autoreset=True)
+
+parser = argparse.ArgumentParser(description="Deletes S3 buckets.")
```

Comment:
In your PR comment you wrote that it doesn't require parameters - what did you mean with that?

Reply:
This is my bad. When I originally wrote the PR, I was using the AWS CLI to configure these parameters and forgot to change the line that says  it doesn't need any when I switched to argparse.